galaxykit/client.py

```diff
@@ -379,3 +380,13 @@ def server_version(self):
         if self._server_version is None:
             self._server_version = self._get_server_version()
         return self._server_version
+
+    @property
+    def ui_endpoint_prefix(self):
```

Comment:
this should probably be named ui_ee_endpoint prefix or something like that?

Not all of _ui/v1 moved here .. and not everything moves to the same place.. for example https://github.com/ansible/ansible-hub-ui/pull/1974 moves things from `_ui/v1/` to `v3/plugin/ansible` and does futher changes to the rest of the url.

Reply:
ah this is a good detail, I wasn't aware that it was more complex. I'll dev a fix - I'd like to keep the API for determining backwards-compatibility changes relatively small and simple.

Reply:
I elected to follow your suggestion to just rename - if we end up with way too many different `_ui_<descriptor>_endpoint_prefix` options, we can come up with something more dry if it's necessary. Until then just renaming is good :)